test/index.js

```diff
@@ -116,7 +115,7 @@ test('Grunt-localizr', function (t) {
         });
     });
     t.on('end', function() {
-        process.chdir(original);
+        process.chdir(path.join(__dirname, '..'));
```

Comment:
I don't follow. `__dirname` doesn't care about cwd. It only cares about the directory where the current file is located.